src/webargs/falconparser.py

```diff
@@ -13,6 +13,9 @@
 status_map = {422: HTTP_422}
 
 
+FALCON_VERSION = falcon.__version__[0]
+
```

Comment:
We could copy compat.py and do

FALCON_VERSION_INFO = tuple(LooseVersion(falcon.__version__).version)

but I don't think there is much interest in doing so.

Let's keep it as is.